<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BewohnerProtokoll</class>
 <widget class="QScrollArea" name="BewohnerProtokoll">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScrollArea</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="widgetResizable">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="scrollAreaWidgetContents">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1189</width>
     <height>777</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,3,0">
    <item row="0" column="1" rowspan="2">
     <widget class="QTextBrowser" name="ProtokollText">
      <property name="autoFormatting">
       <set>QTextEdit::AutoNone</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Protokolldatum:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="ProtokollDatum">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column/>
        <column/>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addTeilnehmer">
        <property name="text">
         <string>neuer Teilnehmer</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/costumer/Teilnehmer</normaloff>:/costumer/Teilnehmer</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Protokolle:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="ProtokollListe"/>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Neues Protokoll</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Datum:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateTimeEdit" name="NewProtokollDatum">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="NewProtokollButton">
           <property name="text">
            <string>Anlegen</string>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/actions/plus</normaloff>:/actions/plus</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
